vipps-checkout-api.md

```diff
@@ -1,52 +1,115 @@
-# Vipps Checkout guide
+# Vipps Checkout
 
-Vipps Checkout is designed to be a low friction, low complexity flow where Vipps Checkout ensures a smooth and efficient checkout experience using the trusted Vipps technology and brand.
-
-Preliminary documentation. Subject to change
-
-API version: 2.0.0.
```

Comment:
I think you need to keep the version numbers, because it's consistent with all the other repos.